src/client/components/checkout/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { Button, ButtonProps, LinkButton } from '../buttons'
+import { PaymentInfo } from './PaymentInfo'
+
+const { white, purple500, gray900 } = colorsV3
+
+export type Props = {
+  buttonText: string
+  buttonOnClick?: () => void
+  buttonLinkTo?: string
+}
+type FooterButtonProps = {
+  background: ButtonProps['background']
+  foreground: ButtonProps['foreground']
+}
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 5rem;
+  padding: 1rem;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  background-color: ${white};
+  box-shadow: 0px -4px 8px rgba(0, 0, 0, 0.05),
+    0px -8px 16px rgba(0, 0, 0, 0.05);
+`
+const InnerWrapper = styled.div`
+  width: 100%;
+  max-width: 628px;
```

Comment:
nit: do we have a constant for this?

Reply:
No, not yet. Have thought about and will add that in another PR :)